src/frontend/komponenter/Felleskomponenter/Hendelsesoversikt/BrevModul/Brevskjema.tsx

```diff
@@ -235,10 +234,24 @@ const Brevskjema = ({ onSubmitSuccess }: IProps) => {
                             )}`}
                         </option>
                     )}
-                </FamilieSelect>
+                </StyledFamilieSelect>
                 <StyledFamilieSelect
                     {...skjema.felter.brevmal.hentNavInputProps(skjema.visFeilmeldinger)}
-                    label={'Velg brevmal'}
+                    label={
+                        <LabelOgEtikett>
+                            <Label
+                                htmlFor={
+                                    skjema.felter.brevmal.hentNavInputProps(skjema.visFeilmeldinger)
+                                        .id
+                                }
```

Comment:
Av ren nysgjerrighet: hva gjør den htmlFor-greia her egentlig?

Reply:
Den gjer at for-property blir satt på Label-elementet. Slik at Label peiker på riktig input.